src/app/common/hooks/use-drawers.ts

```diff
@@ -6,12 +6,14 @@ import {
   useShowSignOut,
   useShowTxSettingsCallback,
   useShowEditNonceState,
+  useShowThemeSwitcherStore,
 } from '@app/store/ui/ui.hooks';
 
 export function useDrawers() {
   const [showAccounts, setShowSwitchAccountsState] = useShowSwitchAccountsState();
   const [showHighFeeConfirmation, setShowHighFeeConfirmation] = useShowHighFeeConfirmationState();
   const [showNetworks, setShowNetworks] = useShowNetworksStore();
+  const [isShowingThemeSwitcher, setIsShowingThemeSwitcher] = useShowThemeSwitcherStore();
```

Comment:
Broke away from the naming convention of the other boolean values and use prefix `is*` to emphasize its boolean type.

Reply:
Not married to `show` but not sure how that doesn't imply a boolean?

Reply:
Agree is prefixed naming convention is better. Can you rename them all for consistency in another commit?

Reply:
@kyranjamie opened this [issue](https://github.com/hirosystems/stacks-wallet-web/issues/2779) to keep track of it. Want to first finish ironing out the code related to this feature.